native/src/main/java/io/ballerina/stdlib/graphql/runtime/observability/GraphqlObservabilityConstants.java

```diff
@@ -0,0 +1,17 @@
+package io.ballerina.stdlib.graphql.runtime.observability;
+
+public class GraphqlObservabilityConstants {
+    public static final String IS_MAIN_SERVICE = "isMainServiceObject";
+    public static final String KEY_INTERCEPTOR = "__interceptor__";
+
+    // Runtime related constants
+    public static final String OBJECT_NAME = "graphql";
+    public static final String CONNECTOR_NAME = "graphql";
+    public static final String QUERY_OPERATION = "query";
+    public static final String MUTATION_OPERATION = "mutation";
+    public static final String SUBSCRIPTION_OPERATION = "subscription";
+
+    // Runtime related tags
+    public static final String TAG_LISTENER_NAME = "listener.name";
+    public static final String TAG_KEY_OPERATION_TYPE = "graphql.operation.type";
```

Comment:
Having a separate class for constants is considered anti-pattern. Shall we move the constants to the `ObserverContext` class?

Reply:
@ThisaruGuruge These constants will be used in `GraphqlObserverContext` as well as in `Engine`. Is it okay to move them into `GraphqlObserverContext`?

Reply:
Yes, that is okay.